libcpp/charset.cc

```diff
@@ -1256,6 +1256,38 @@ _cpp_uname2c_uax44_lm2 (const char *name, size_t len, char *canon_name)
   return result;
 }
 
+/* Returns 1 if C has the XID_Start property, 2 if C has the XID_Continue properties, 3 if C has the both properties, 0 otherwise. */
+int check_xid_property (cppchar_t c) {
+  // fast path for ASCII
+  if (c < 0x80) {
+    if (('A' <= c && c <= 'Z') || ('a' <= c && c <= 'z'))
+      return 3;
+    if (('0' <= c && c <= '9') || c == '_')
+      return 2;
+  }
```

Comment:
I think this function should return bit flags but how can I do it?
(i.e. XID_Start: 1, XID_Continue: 1 << 1)
Is just using `int` (or `unsigned int`) and export constants such as `XID_START` and `XID_CONTINUE` good?

Reply:
Fix to use `unsigned int` and export consants via enum